src/components/chart/element/element.tip.js

```diff
@@ -1,5 +1,5 @@
-import { numberWithComma } from '@/common/utils';
 import dayjs from 'dayjs';
+import { numberWithComma } from '@/common/utils';
 import Canvas from '../helpers/helpers.canvas';
 import { truthyNumber } from '../../../common/utils';
```

Comment:
`numberWithComma` 가 위치한 `@/common/utils` 경로랑 `truthynumber` 가 위치한 `../../../common/utils` 가 같은 경로로 보여지는데 하나로 합쳐주시면 더 좋을 것 같습니다~~

```suggestion
import { numberWithComma, truthyNumber } from '@/common/utils';
import Canvas from '../helpers/helpers.canvas';
```

Reply:
알려주셔서 감사합니다 같은 경로 하나로 합쳤습니다!!